main/templates/main/index.html

```diff
@@ -3,78 +3,59 @@
+          a ♥ if you like it!
         </p>
-        <p>
-        {%for source in data_sources%}
-          <a href="{%url 'data-sources' source%}" class="btn btn-primary btn-primary-hollow btn-lg padded">
-          {{source}}
-        </a>
-        {%endfor%}
-        </p>
-        <a
-          {% if client_id %}
-          class="btn btn-primary btn-lg"
-          href="https://www.openhumans.org/direct-sharing/projects/oauth2/authorize/?client_id={{ client_id }}&response_type=code&redirect_uri={{ redirect_uri }}"
-          {% else %}
-          class="btn btn-default disabled"
-          {% endif %}
-          >
-          Get started
+        <hr>
+        {% include 'main/partials/notebooks-table.html' %}
+        <hr>
+        <p class="small">
+          Want to add a notebook to the gallery?
+        {% if client_id %}
+        <a href="https://www.openhumans.org/direct-sharing/projects/oauth2/authorize/?client_id={{ client_id }}&response_type=code&redirect_uri={{ redirect_uri }}">
+          Click here to log in
         </a>
-        {% if not client_id %}
+        {% else %}
         <p class="text-muted"><i>
-          This button is disabled because client_id hasn't been configured.
+          The log in link is unavailable because client_id hasn't been configured.
         </i></p>
         {% endif %}
-      </div>
-      <div class='col-md-6'>
-        <h2>The latest notebook additions</h2>
-        Check out the latest notebooks that have been shared by our community.
-        <div class="table-responsive">
-        <table class='table'>
-          <thead>
-            <tr>
-              <th>Title</th>
-              <th>Data sources</th>
-              <th>Updated</th>
-            </tr>
-          </thead>
-          <tbody>
-            {%for notebook in latest_notebooks%}
-            <tr>
-              <td>
-                <a href="{%url 'notebook-details' notebook.id%}">
-                  <code>{{notebook.notebook_name}}</code>
-                  <small>by {{notebook.oh_member.oh_username}} </small><i class="fas fa-info-circle"></i>
-                </a>
-              </td>
-              <td>
-                {%for source in notebook.get_data_sources_json%}
-                <a href="{%url 'search' %}?search_term={{source}}&search_field=data_sources">
-                <span class="label label-info">{{source}}</span>
-                </a>
-                {%endfor%}
-              </td>
-              <td>{{notebook.updated_at|timesince}} ago</td>
-            </tr>
-            {%endfor%}
-          </tbody>
-        </table>
-        </div>
-      </div>
-    </div>
+      </div> <!-- /.col-xs-12 -->
+    </div> <!-- /.row -->
 </div>
 {% endblock %}
+
+{% block extra_js %}
+<script src="https://cdnjs.cloudflare.com/ajax/libs/list.js/1.5.0/list.min.js"></script>
+
+<script>
+  var options = {
+    valueNames: [ 'name', 'source', 'date' ]
+  };
+
+  {% for source in data_sources %}
```

Comment:
This is a bit problematic as is, as the `data_sources` are created over all existing notebooks, but the table only shows the last 5 notebooks I think. So there can be `data_sources` in the head which are not displayed in the table.

Reply:
Ah, sorry, I didn't realize the table was limited, I didn't look closely at the view – this modification definitely envisions a bigger starting table.